pkg/konfluxgen/fbc-builder.yaml

```diff
@@ -125,61 +116,16 @@ spec:
     value: $(tasks.clone-repository.results.commit)
   tasks:
   - name: fbc-related-image-check
-    runAfter:
```

Comment:
How can this work? Just listing a task without referencing `taskRef`.
Same for the build-container task below. Was this removed by accident?

If I understand it correctly this "fbc-related-image-check" task is now merged into "validate-fbc" task that you're adding in this PR. So we can probably remove this fbc-related-image-check.
Also, the parameter "skip-fbc-related-image-check" for skipping this task in pre-submit checks won't work. We can probably remove it completely because skipping the whole "validate-fbc" is probably not a good thing.

Reply:
- name: fbc-related-image-check
    when:
    - input: $(params.skip-fbc-related-image-check)
      operator: in
      values:
      - "false"
      
Removing line 118 to 123 will work? @mgencur 
and rest of the changes are done, is there any change required @pierDipi @creydr

Reply:
> Removing line 118 to 123 will work?
It should work. I don't see how it can work in the current form. But we'll need to test!

Reply:
The `fbc-related-image-check` task seems not to be included in the new fbc multiarch pipeline. So I guess we can remove the `patch_related_image_check.patch`.
Also the `patch_build_args.patch` seems not to be needed anymore for the FBC, as we have this in the new FBC pipeline: https://github.com/openshift-knative/hack/blob/f69466d0e07e832521daf106a3ed1780fe8d67b2/pkg/konfluxgen/fbc-builder.yaml#L239-L241